src/docs/ftw-introduction/ftw-product/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: FTW product
+slug: ftw-product
+updated: 2021-03-11
+category: ftw-introduction
+ingress:
+  This article introduces FTW-product and what's different compared to
+  FTW-daily and FTW-hourly.
+published: true
+---
+
+FTW-product is a new Flex Template for Web. It's build with product
+marketplaces in mind.
+
+## Differences in directory structure
+
+With this template, we decided to take more out of the possibilities
+that code-splitting made possible. FTW-daily and FTW-hourly were created
+before code-splitting was possible, and pushing most of the components
+to a specific components directory (_src/components/_) made sense. It
+was a clear place where customizers could start looking for a components
+they wanted to customize. This setup also improved developer experience
+a bit since it allowed us to use index file to export all those
+components:
+
+```js
+import { Avatar, Button, Logo } from ‘../../components’;
+```
+
+The alternative is to import everything one-by-one, which can be a
+pretty long list of imports.
+
+```js
+Import Avatar from ‘../../components/Avatar/Avatar.js’;
+Import Button from ‘../../components/Button/Button.js’;
+Import Logo from ‘../../components/Logo/Logo.js’;
+```
+
+Code-splitting changed the dynamics of this setup a bit. Everything that
+is imported through a single file, is likely to end up into a main
+code-chunk. That means that everything in that file is going to slow
+down page rendering - when the page is fully loaded. That also affects
+search engine optimization (SEO).
+
+With FTW-product, we have decided to move certain page-specific files
+under page-directories. Here are some of the relevant changes in the
+directory structure:
+
+- **src/config/**: Generic configuration files are moved to this
+  directory
+  - Extra configs:
+    - **mainSearchType** in Topbar. It's either _'keywords'_ or
+      _'location'_.
+    - **searchPageVariant**: _'list'_ & _'map'_.
+    - **listingPageVariant**: _'product'_ & 'TODO: hero'
+  - Renamed configs:
+    - **transactionProcessAlias** (in other templates:
+      bookingProcessAlias)
+    - **lineItemUnitType** (in other templates: bookingUnitType)
+- **src/routing**: Route configuration and UI components related to
+  routing are moved to this directory
+- UI components that are only used in a single page or within a more
+  complex component are nested within that page or component. For
+  example:
+  - _EditListinWizard_ is moved under _EditListinPage_
+  - Search map and filters are moved under _SearchPage_
+  - _TransactionPage_ got subcomponents like _ActivityFeed_,
+    _ReviewModal_, _ReviewForm_, _SendMessageForm_, and
+    _TransactionPanel_.
+- **src/forms**: this directory was removed as there was only one form
+  that was truly shared between pages: _StripeConnectAccountForm_. It's
+  moved to _src/components/_.
+
+TODO: add other notable changes here
```

Comment:
Do we need to mention that the booking breakdown is now handled by the more generic `OrderPanel` that renders the new `ProductOrderForm` or the old `BookingDatesForm`?